config/server_start.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 
-DIR="$(dirname "${BASH_SOURCE[0]}")"
+# DIR="$(dirname "${BASH_SOURCE[0]}")"
 
 # Run web application
-exec newrelic-admin run-program gunicorn -c "$DIR/gunicorn.conf.py" --worker-class gevent --paste $CKAN_INI "$@" --timeout 120 --workers 2
+# exec newrelic-admin run-program gunicorn -c "$DIR/gunicorn.conf.py" --worker-class gevent --paste $CKAN_INI "$@" --timeout 120 --workers 2
+
+# start xloader
+exec ckan -c $CKAN_INI jobs worker &
+# Fire it up!
+exec ckan -c $CKAN_INI run --host "0.0.0.0" --port $PORT
```

Comment:
This doesn't work yet, right?

Reply:
This does, we utilized this to start up the development space for inventory on cloud.gov